scripts/evaluate_predicate_classifiers.py

```diff
@@ -1,19 +1,23 @@
 """Script to evaluate learned predicate classifiers on held-out test cases."""
 
-from typing import List, Set
+from typing import Callable, List, Set
 
 import numpy as np
 
 from predicators.src import utils
 from predicators.src.approaches import create_approach
-from predicators.src.approaches.bilevel_planning_approach import \
-    BilevelPlanningApproach
+from predicators.src.approaches.interactive_learning_approach import \
+    InteractiveLearningApproach
 from predicators.src.envs import BaseEnv, create_new_env
```

Comment:
Since we are now committing to the interactive learning approach here and in the other script, which I think is fine, let's rename the files and update the file docstrings to make this clear.